content/page/liste-speaker.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Liste Speaker"
+date: 2022-11-30T12:39:45+01:00
+draft: false
+---
+
+# Speaker.ine.s confirmé.e.s pour la track onestla.tech aux ApiDays Paris 2022
+
+- [**Marion Agé**](https://twitter.com/marion_age) : cogérante et CTO de la coopérative Les-Tilleuls.coop.
+- **Cécile Staehle** : Coopératrice en charge du développement RH chez Troopers.
```

Comment:
"coopératrice", peut-être mettre le lien vers son profil Linkedin : https://www.linkedin.com/in/c%C3%A9cile-staehle/

Reply:
C'est juste un sentiment, parceque les deux types de profils sont publics, mais ça m'embêterait plus qu'on link mon LinkedIn sans me demander que mon Twitter. Mais c'est vrai qu'un lien serait cool